test/specs/mainnet/exchangeRate/Smoke_ExchangeRate.spec.js

```diff
@@ -0,0 +1,188 @@
+        responsesCoidList = await axios.get(
+          "https://api.coingecko.com/api/v3/coins/list?include_platform=true"
+        );
+
+        for (let z = 0; z < responsesCoidList.data.length; z++) {
+          if (
+            typeof responsesCoidList.data[z].platforms[network_coinGecho[n]] ===
+              "string" &&
+            responsesCoidList.data[z].platforms[network_coinGecho[n]] != ""
+          ) {
+            TOKEN_LIST_ADDRESS_COINGECHO.push(
+              responsesCoidList.data[z].platforms[network_coinGecho[n]]
+            );
+            TOKEN_LIST_ID_COINGECHO.push(responsesCoidList.data[z].id);
+          }
+        }
+      } catch (e) {
+        assert.fail(
+          "An error is displayed while getting the token addresses and it's rate from the CoinGecho."
+        );
+      }
+
+      try {
+        // get the token list of the Etherspot
+        let TokenDetails = await mainNetSdk.getTokenListTokens({
+          name: "EtherspotPopularTokens",
+        });
+
+        // get the list of token address of the Etherspot
+        for (let x = 0; x < TokenDetails.length; x++) {
+          TOKEN_LIST_ADDRESS_ETHERSPOT.push(TokenDetails[x].address);
+        }
+
+        // get the chain id of the Etherspot
+        XDAI_CHAIN_ID_ETHERSPOT = TokenDetails[0].chainId;
+
+        // Request payload for fetch the token rates informaiton of the Etherspot
+        requestPayload = {
+          tokens: TOKEN_LIST_ADDRESS_ETHERSPOT,
+          chainId: XDAI_CHAIN_ID_ETHERSPOT,
+        };
+      } catch (e) {
+        assert.fail(
+          "An error is displayed while fetching the rate of the token from Etherspot."
+        );
+      }
+
+      // Fetch the token rates of the Etherspot and compare with coinGecho
+      try {
+        let rates = await mainNetSdk.fetchExchangeRates(requestPayload);
+        for (let y = 0; y < rates.items.length; y++) {
+          for (let j = 0; j < TOKEN_LIST_ADDRESS_COINGECHO.length; j++) {
+            Helper.wait(4000);
+            let etherspotAddress = rates.items[y].address;
+            console.log("Etherspot Address:", etherspotAddress.toLowerCase());
+            console.log("CoinGecho Address:", TOKEN_LIST_ADDRESS_COINGECHO[j]);
+            if (
+              etherspotAddress.toLowerCase() == TOKEN_LIST_ADDRESS_COINGECHO[j]
+            ) {
+              Helper.wait(4000);
+
+              let responsesCoidMarket = await axios.get(
+                "https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&ids=" +
+                  TOKEN_LIST_ID_COINGECHO[j] +
+                  "&per_page=100&page=1&sparkline=true&locale=en"
+              );
+
+              let num1 = responsesCoidMarket.data[0].current_price; // rates of the coingecho
+              let num2 = rates.items[y].usd; // rates of the etherspot
+
+              let deviationPercentage =
+                (Math.abs(num1 - num2) / ((num1 + num2) / 2)) * 100;
+              if (deviationPercentage > 5) {
```

Comment:
Please make the deviation percentage a top level variable as we may need to change this later.

Reply:
Modified